applications/openshift/master/file_owner_ovn_cni_server_sock/tests/ocp4/e2e.yml

```diff
@@ -0,0 +1,3 @@
+---
+default_result: PASS
```

Comment:
This rules results in NOT-APPLICABLE due to ovn platform:
https://github.com/ComplianceAsCode/content/blob/9613cee7adba0d786e6b9131ef2447394362a153/applications/openshift/master/file_owner_ovn_cni_server_sock/rule.yml#L4

Reply:
@rhmdnd I think the default result for this rule should be `NOT-APPLICABLE`

In [latest](https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/test-platform-results/pr-logs/pull/ComplianceAsCode_content/11759/pull-ci-ComplianceAsCode-content-master-e2e-aws-ocp4-pci-dss-node/1779827610050629632/artifacts/e2e-aws-ocp4-pci-dss-node/test/build-log.txt) and [4.15](https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/test-platform-results/pr-logs/pull/ComplianceAsCode_content/11759/pull-ci-ComplianceAsCode-content-master-4.15-e2e-aws-ocp4-pci-dss-node/1779827609874468864/artifacts/e2e-aws-ocp4-pci-dss-node/test/build-log.txt)  we got `NOT-APPLICABLE`

[4.13](https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/test-platform-results/pr-logs/pull/ComplianceAsCode_content/11759/pull-ci-ComplianceAsCode-content-master-4.13-e2e-aws-ocp4-pci-dss-node/1779827609731862528/artifacts/e2e-aws-ocp4-pci-dss-node/test/build-log.txt) indeed resulted in `PASS`, similar to [4.12](https://github.com/ComplianceAsCode/content/blob/bc744741473babf8be5c7ba2d3bb1ecddfdaad9d/applications/openshift/master/file_owner_ovn_cni_server_sock/tests/ocp4/4.12.yml)
https://github.com/ComplianceAsCode/content/blob/bc744741473babf8be5c7ba2d3bb1ecddfdaad9d/applications/openshift/master/file_owner_ovn_cni_server_sock/tests/ocp4/4.12.yml